catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -164,14 +166,23 @@ public UpdateStorageResponse version(UpdateStorageRequest streamUpdateRequest,
         setSkipFlag(streamUpdateRequest);
         setSkipFlag(updateStorageResponse);
 
-        Collection<ReadStorageRequest> ids = getReadStorageRequests(updateStorageResponse);
+        List<Metacard> updatedMetacards = updateStorageResponse.getUpdatedContentItems()
+                .stream()
+                .filter(ci -> ci.getQualifier() == null || ci.getQualifier()
+                        .equals(""))
+                .map(ContentItem::getMetacard)
+                .filter(isVersionOrDeleted().negate())
+                .filter(Objects::nonNull)
```

Comment:
yes we should! and you reminded me to go update the method that checks if its that type (mentioned in your comment below) that I should be checking for null there as well.
fixed.

Unless you think we should get rid of this explicit null check since [null] is now handled.